src/app/tests/TestReadInteraction.cpp

```diff
@@ -151,6 +154,16 @@ class MockInteractionModelApp : public chip::app::InteractionModelDelegate
         return err;
     }
 
+    void OnReportData(const chip::app::ReadClient * apReadClient, const chip::app::ClusterInfo & aPath,
+                      chip::TLV::TLVReader * apData, chip::Protocols::InteractionModel::ProtocolCode status) override
+    {
+        mGotAttributeResponse = true;
+    }
+    CHIP_ERROR ReportProcessed(const chip::app::ReadClient * apReadClient) override
```

Comment:
supernit: add one empty line separator between methods for consistency. Above and below.

Reply:
Done, thanks